public/app/plugins/datasource/influxdb/components/InfluxLogsQueryField.tsx

```diff
@@ -12,7 +12,7 @@ import { InfluxQuery, InfluxOptions } from '../types';
 export interface Props extends ExploreQueryFieldProps<InfluxDatasource, InfluxQuery, InfluxOptions> {}
 
 export interface State {
-  measurements: CascaderOption[];
+  measurements: ButtonCascaderOption[];
```

Comment:
Why `ButtonCascader` needs a different type for options? I suggest having a single type for that as options should have the same type regardless of the control element

Reply:
I didn't know which type to use when I started. Turns out this is the same type exported form rc-cascader. I'll change so we have the same type for everything.